### PR TITLE
Convert Base64 to Kotlin

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

File: okio/src/test/java/okio/DeflateKotlinTest.kt [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 package okio
 
+import okio.ByteString.Companion.decodeHex
```
Comment (swankjesse) [inline okio/src/test/java/okio/DeflateKotlinTest.kt]: This doesn’t seem right. Any advice? It yields the right Java API.

File: okio-kotlin/src/main/java/okio/ByteString.kt [hunk 2]
```diff
@@ -46,20 +46,6 @@ inline fun ByteBuffer.toByteString(): ByteString = ByteString.of(this)
...
- * @see ByteString.decodeBase64
- */
-inline fun String.decodeBase64(): ByteString? = ByteString.decodeBase64(this)
-
-/**
- * Returns a [ByteString] decoded from this hex [String].
- *
- * @see ByteString.decodeHex
- */
-inline fun String.decodeHex(): ByteString = ByteString.decodeHex(this)
```
Comment (JakeWharton) [inline okio-kotlin/src/main/java/okio/ByteString.kt]: Delete/move the tests specific to this?

File: okio/src/test/java/okio/DeflateKotlinTest.kt (hunk 1 above)
Comment (swankjesse) [reply okio/src/test/java/okio/DeflateKotlinTest.kt]: screen shot 2018-04-25 at 3 27 13 pm

Comment (swankjesse) [reply okio/src/test/java/okio/DeflateKotlinTest.kt]: not yet

Comment (JakeWharton) [reply okio/src/test/java/okio/DeflateKotlinTest.kt]: Ah, yeah. We went through this on KotlinPoet. Sigh.

File: okio-kotlin/src/main/java/okio/ByteString.kt (hunk 2 above)
Comment (swankjesse) [reply okio-kotlin/src/main/java/okio/ByteString.kt]: Done.

**Pull request merged**